.github/workflows/ci.yml

```diff
@@ -238,6 +238,7 @@ jobs:
       - name: Install Linux dependencies
         if: startsWith(matrix.os, 'ubuntu')
         run: |
+          sudo apt-get update
```

Comment:
Is this sufficient, or should I cherry-pick  #4356?

Reply:
it can be done with cherry-pick and minor update. so please cherry-pick.